src/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -0,0 +1,237 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <ProjectGuid>{2D694AC8-A12F-4622-9405-74E20EC40C3A}</ProjectGuid>
+    <RootNamespace>System.ComponentModel.Composition</RootNamespace>
+    <AssemblyName>System.ComponentModel.Composition</AssemblyName>
+    <DefineConstants>$(DefineConstants);FEATURE_TRACING</DefineConstants>
+    <!-- Workaround for issue buildtools #1784 -->
+    <RunApiCompat>false</RunApiCompat>
+  </PropertyGroup>
+  <!-- Default configurations to help VS understand the options -->
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.0-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.0-Release|AnyCPU'" />
+  <ItemGroup>
+    <Reference Include="Microsoft.Win32.Primitives" />
```

Comment:
I believe this whole References should go down to the itemgroup that has the condition of `$(TargetGroup) != 'netfx'`

Reply:
I just took out the netfx stuff as it's not a configuratoin.